ironic/dnsmasq.conf

```diff
@@ -16,3 +16,7 @@ dhcp-boot=tag:efi,tag:!ipxe,ipxe.efi
 
 # Client is running PXE over BIOS; send BIOS version of iPXE chainloader
 dhcp-boot=/undionly.kpxe,172.22.0.1
+
+# Disable DHCP and DNS over provisioning network
+dhcp-option=3
+dhcp-option=6
```

Comment:
with option 6 should we not pass additional DNS server(s)?

Reply:
this is so that ironic's dnsmasq does not set DNS servers (just passing 6 is equivalent to nulling it).